packages/client/hmi-client/src/services/goLLM.ts

```diff
@@ -51,6 +51,33 @@ export async function interventionPolicyFromDocument(
 	return data;
 }
 
+export async function enrichModelMetadata(modelId: string, documentId: string, overwrite: boolean): Promise<void> {
+	try {
+		const response = await API.get<TaskResponse>('/gollm/enrich-model-metadata', {
+			params: {
+				'model-id': modelId,
+				'document-id': documentId,
+				overwrite
+			}
+		});
+
+		const taskId = response.data.id;
+		await handleTaskById(taskId, {
+			ondata(data, closeConnection) {
+				if (data?.status === TaskStatus.Failed) {
+					closeConnection();
+					throw new FatalError('Task failed');
+				}
+				if (data.status === TaskStatus.Success) {
+					closeConnection();
+				}
+			}
+		});
```

Comment:
I think we can remove this code. Since, instead of creating connection for each task, we can subscribe/unsubscribe on global sse event channel whenever needed using `subscribe` in `ClientEventService.ts` or `useClientEvent.ts`

Reply:
And I believe the endpoint for subscribing for each individual task event got removed in the server side at some point and this code won't work.